plugins/modules/identity/ipa/ipa_otptoken.py

```diff
@@ -0,0 +1,530 @@
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: ipa_otptoken
+author: justchris1 (@justchris1)
+short_description: Manage FreeIPA OTPs
+version_added: 2.5.0
+description:
+- Add, modify, and delete One Time Passwords in IPA.
+options:
+  uniqueid:
+    description: Unique ID of the token in IPA.
+    required: True
+    aliases: ["name"]
+    type: str
+  newuniqueid:
+    description: If specified, the unique id specified will be changed to this.
+    type: str
+  otptype:
+    description:
+    - Type of OTP.
+    - "B(Note:) Cannot be modified after OTP is created."
+    type: str
+    choices: [ totp, hotp ]
+  secretkey:
+    description:
+    - Token secret (Base64).
+    - If OTP is created and this is not specified, a random secret will be generated by IPA.
+    - "B(Note:) Cannot be modified after OTP is created."
+    type: str
+  description:
+    description: Description of the token (informational only).
+    type: str
+  owner:
+    description:  Assigned user of the token.
+    type: str
+  enabled:
+    description: Mark the token as enabled (default C(true)).
+    default: true
+    type: bool
+  notbefore:
+    description:
+    - First date/time the token can be used.
+    - In the format C(YYYYMMddHHmmss).
+    - For example, C(20180121182022) will allow the token to be used starting on 21 January 2018 at 18:20:22.
+    type: str
+  notafter:
+    description:
+    - Last date/time the token can be used.
+    - In the format C(YYYYMMddHHmmss).
+    - For example, C(20200121182022) will allow the token to be used until 21 January 2020 at 18:20:22.
+    type: str
+  vendor:
+    description: Token vendor name (informational only).
+    type: str
+  model:
+    description: Token model (informational only).
+    type: str
+  serial:
+    description: Token serial (informational only).
+    type: str
+  state:
+    description: State to ensure.
+    choices: ['present', 'absent']
+    default: 'present'
+    type: str
+  algorithm:
+    description:
+    - Token hash algorithm.
+    - "B(Note:) Cannot be modified after OTP is created."
+    choices: ['sha1', 'sha256', 'sha384', 'sha512']
+    type: str
+  digits:
+    description:
+    - Number of digits each token code will have.
+    - "B(Note:) Cannot be modified after OTP is created."
+    choices: [ 6, 8 ]
+    type: int
+  offset:
+    description:
+    - TOTP token / IPA server time difference.
+    - "B(Note:) Cannot be modified after OTP is created."
+    type: int
+  interval:
+    description:
+    - Length of TOTP token code validity in seconds.
+    - "B(Note:) Cannot be modified after OTP is created."
+    type: int
+  counter:
+    description:
+    - Initial counter for the HOTP token.
+    - "B(Note:) Cannot be modified after OTP is created."
+    type: int
+extends_documentation_fragment:
+- community.general.ipa.documentation
+
+requirements:
+- base64
```

Comment:
I don't think this should be mentioned, since `base64` has been part of the Python standard library since Python 2.4.

Reply:
I will remove it then.